### PR TITLE
[MRG] - Updated GcRealityManagerData

### PR DESCRIPTION
**Pull request by HolterPhylo**

### CODE REVIEW

**Comment by HolterPhylo** [comment]
Compiles ok. 
De-re-compiles ok BACKUPREALITY.MBIN, no errors in log. Binaries compares ok
De-re-compiles ok DEFAULTREALITY.MBIN, no errors in log.. Binaries show 2 differences. The only thing I can think of is that these errors are due to hex arithmetic!

File: libMBIN/Source/NMS/GameComponents/GcRealityManagerData.cs [hunk 1]
```diff
@@ -50,6 +50,10 @@ public class GcRealityManagerData : NMSTemplate
         /* 0x0818 */ public List<NMSString0x80> AlienPuzzlesTables;
         [NMS(Size = 0x65, EnumType = typeof(GcInteractionType.InteractionTypeEnum))]
         /* 0x0828 */ public bool[] LoopInteractionPuzzles;
+
+        //[NMS(Size = 0x3, Ignore = true)]
```
Comment (monkeyman192) [inline libMBIN/Source/NMS/GameComponents/GcRealityManagerData.cs]: why is this added but commented out?

Comment (HolterPhylo) [reply libMBIN/Source/NMS/GameComponents/GcRealityManagerData.cs]: You said before it is not technically needed but since I don't know exactly how those rules work yet, I did try it out both ways (commented/commented out).  Works both ways.  Left it there as a reminder.

**Comment by monkeyman192** [comment]
> 
> 
> Compiles ok.
> De-re-compiles ok BACKUPREALITY.MBIN, no errors in log. Binaries compares ok
> De-re-compiles ok DEFAULTREALITY.MBIN, no errors in log.. Binaries show 2 differences. The only thing I can think of is that these errors are due to hex arithmetic!

Can you show a picture of the binary differences?

File: libMBIN/Source/NMS/GameComponents/GcRealityManagerData.cs (hunk 1 above)
Comment (monkeyman192) [reply libMBIN/Source/NMS/GameComponents/GcRealityManagerData.cs]: maybe pick to keep it or not. Leaving it in as comment it pointless.
In this case, as you have noticed, it won't matter if it is there or not

Comment (HolterPhylo) [reply libMBIN/Source/NMS/GameComponents/GcRealityManagerData.cs]: removed commented lines

**Comment by HolterPhylo** [comment]
1) First diff, one byte:  image

2)  Second diff, 4 bytes spacing. Starts here:  
image

Ends here: 
image

After that, no more diff.  Same length:  
image

**Comment by monkeyman192** [comment]
1. This is a list header to the `MissionDetailIcons` data in the `GcRealityIconTable Icons` field in this struct.
This pointer points to the location you have showed in picture 2.
If I were to guess, the solution would be to give `GcRealityIcon` an alignment value of 0x8 (try this change and see if it works.)

**Comment by HolterPhylo** [comment]
Perfect! And learned that 0x6440 was an offset...
Hex compare is 100% ok, both MBINs

I lurked around that ``GcRealityIcon.cs`` so many times, but could not figure what it was...
